draco/schema.py

```diff
@@ -46,25 +78,42 @@ def schema_from_dataframe(
         entropy = -(vc * np.log(vc) / np.log(e)).sum()
         entropy = round(entropy * 1000)
 
-        props = {"name": col, "type": data_type, "unique": unique, "entropy": entropy}
-
-        if data_type == "number":
-            props["min"] = int(column.min())
-            props["max"] = int(column.max())
-            props["std"] = int(column.std())
-
-        elif data_type == "string":
-            objcounts = column.value_counts()
-            props["freq"] = objcounts.iloc[0]
-
+        props: FieldProps = _construct_field_props(
+            col, data_type, unique, entropy, column
+        )
         schema["field"].append(props)
 
     return schema
 
 
-def schema_from_file(
-    file_path: Path, parse_data_type=dtype_to_field_type
-) -> Dict[str, Any]:
+def _construct_field_props(
+    name: str,
+    ty: FieldType,
+    unique: int,
+    entropy: float,
```

Comment:
Why pass in these two and compute the others?

Reply:
Given that these values were included in a column's properties regardless of its data type, I just kept their computation in `schema_from_dataframe`.  But since we have a `_construct_field_props` function, it is right that it makes more sense to handle all computations there.

I addressed this in 46358c9